components/layout/flow.rs

```diff
@@ -635,9 +625,6 @@ bitflags! {
     pub flags FlowFlags: u32 {
         // text align flags
         #[doc = "Whether this flow must have its own layer. Even if this flag is not set, it might"]
```

Comment:
Thanks for pointing this out! I've fixed it in the latest version of the branch.